pinot-tools/src/main/resources/examples/batch/fineFoodReviews/fineFoodReviews_offline_table_config.json

```diff
@@ -37,7 +37,8 @@
       "name": "Text",
       "properties": {
         "deriveNumDocsPerChunkForRawIndex": "true",
-        "rawIndexWriterVersion": "3"
+        "rawIndexWriterVersion": "3",
+        "caseSensitiveIndex": "false"
```

Comment:
Suggest modifying this to use the new format. `properties` are already deprecated. Same for other tests

Reply:
done